src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -54,14 +61,23 @@ public SqlConnection(string databasePath)
 
             Contract.ThrowIfNull(handle);
 
+            raw.sqlite3_busy_timeout(handle, (int)TimeSpan.FromMinutes(1).TotalMilliseconds);
```

Comment:
:bulb: If this can throw an exception, make sure we close `handle` if necessary since it won't be wrapped in `SqlConnection`.